<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="15" passed="2" failed="3" skipped="10">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-17T12:24:17Z" name="Default suite" finished-at="2025-04-17T12:27:03Z" duration-ms="166327">
    <groups>
    </groups>
    <test started-at="2025-04-17T12:24:17Z" name="Default test" finished-at="2025-04-17T12:27:03Z" duration-ms="166327">
      <class name="Test_Suit.SmokeTestSuite">
        <test-method is-config="true" signature="setUp()[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:24:17Z" name="setUp" finished-at="2025-04-17T12:24:35Z" duration-ms="18799" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="AddGoalPlan()[pri:1, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:24:35Z" name="AddGoalPlan" finished-at="2025-04-17T12:25:15Z" duration-ms="39851" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddGoalPlan -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:25:15Z" name="runOnFailure" finished-at="2025-04-17T12:25:15Z" duration-ms="3" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=AddGoalPlan status=SUCCESS method=SmokeTestSuite.AddGoalPlan()[pri:1, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method signature="AddPMSCycle()[pri:2, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:25:15Z" name="AddPMSCycle" finished-at="2025-04-17T12:25:35Z" depends-on-methods="Test_Suit.SmokeTestSuite.AddGoalPlan" duration-ms="19940" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddPMSCycle -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:25:35Z" name="runOnFailure" finished-at="2025-04-17T12:25:35Z" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=AddPMSCycle status=SUCCESS method=SmokeTestSuite.AddPMSCycle()[pri:2, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method signature="Initiate_PMSCycle()[pri:3, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:25:35Z" name="Initiate_PMSCycle" finished-at="2025-04-17T12:25:44Z" depends-on-methods="Test_Suit.SmokeTestSuite.AddPMSCycle" duration-ms="9041" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitForElementVisiblity(org.openqa.selenium.By)" because "this.ExplicitWait" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitForElementVisiblity(org.openqa.selenium.By)" because "this.ExplicitWait" is null
at NormalFlowForEmployee.initiatePMSCycle.enterWeightages(initiatePMSCycle.java:61)
at Test_Suit.SmokeTestSuite.Initiate_PMSCycle(SmokeTestSuite.java:111)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Initiate_PMSCycle -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:25:44Z" name="runOnFailure" finished-at="2025-04-17T12:25:44Z" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Initiate_PMSCycle status=SKIP method=SmokeTestSuite.Initiate_PMSCycle()[pri:3, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method signature="Initiate_PMSCycle()[pri:3, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:25:44Z" name="Initiate_PMSCycle" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.AddPMSCycle" duration-ms="40334" status="FAIL">
          <exception class="org.openqa.selenium.ElementNotInteractableException">
            <message>
              <![CDATA[element not interactable
  (Session info: chrome=135.0.7049.85)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.6'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [4f3d02a1851196970a609becc82cc147, clickElement {id=f.D722E01BB8E78382691FE6027E00DC7F.d.D8721BB56F58BF66385DD305FDC9EE77.e.797}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.85, chrome: {chromedriverVersion: 135.0.7049.95 (de2eb485a195..., userDataDir: C:\Users\DELL\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:55431}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:55431/devtoo..., se:cdpVersion: 135.0.7049.85, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (4f3d02a1851196970a609becc82cc147)] -> xpath: //a[@ng-click="addNewRow = true"]]
Session ID: 4f3d02a1851196970a609becc82cc147]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementNotInteractableException: element not interactable
  (Session info: chrome=135.0.7049.85)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.6'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [4f3d02a1851196970a609becc82cc147, clickElement {id=f.D722E01BB8E78382691FE6027E00DC7F.d.D8721BB56F58BF66385DD305FDC9EE77.e.797}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.85, chrome: {chromedriverVersion: 135.0.7049.95 (de2eb485a195..., userDataDir: C:\Users\DELL\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:55431}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:55431/devtoo..., se:cdpVersion: 135.0.7049.85, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (4f3d02a1851196970a609becc82cc147)] -> xpath: //a[@ng-click="addNewRow = true"]]
Session ID: 4f3d02a1851196970a609becc82cc147
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at NormalFlowForEmployee.initiatePMSCycle.goToWeightTab(initiatePMSCycle.java:43)
at Test_Suit.SmokeTestSuite.Initiate_PMSCycle(SmokeTestSuite.java:109)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementNotInteractableException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Initiate_PMSCycle -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="runOnFailure" finished-at="2025-04-17T12:26:25Z" duration-ms="389" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Initiate_PMSCycle status=FAILURE method=SmokeTestSuite.Initiate_PMSCycle()[pri:3, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method signature="Manager_AddGoalTo_Emp()[pri:4, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="Manager_AddGoalTo_Emp" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.Initiate_PMSCycle" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestSuite.Manager_AddGoalTo_Emp()[pri:4, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestSuite.Manager_AddGoalTo_Emp()[pri:4, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Manager_AddGoalTo_Emp -->
        <test-method signature="Employee_AssessmentSubmission()[pri:5, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="Employee_AssessmentSubmission" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.Manager_AddGoalTo_Emp" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestSuite.Employee_AssessmentSubmission()[pri:5, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestSuite.Employee_AssessmentSubmission()[pri:5, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Employee_AssessmentSubmission -->
        <test-method signature="Manager_AssessmentSubmissionTo_Skip()[pri:6, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="Manager_AssessmentSubmissionTo_Skip" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.Employee_AssessmentSubmission" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestSuite.Manager_AssessmentSubmissionTo_Skip()[pri:6, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestSuite.Manager_AssessmentSubmissionTo_Skip()[pri:6, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Manager_AssessmentSubmissionTo_Skip -->
        <test-method signature="Skip_Approval()[pri:7, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="Skip_Approval" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.Manager_AssessmentSubmissionTo_Skip" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestSuite.Skip_Approval()[pri:7, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestSuite.Skip_Approval()[pri:7, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Skip_Approval -->
        <test-method signature="OneToOneMeeting_Manager()[pri:8, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="OneToOneMeeting_Manager" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.Skip_Approval" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestSuite.OneToOneMeeting_Manager()[pri:8, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestSuite.OneToOneMeeting_Manager()[pri:8, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OneToOneMeeting_Manager -->
        <test-method signature="OnToOneMeeting_Employee()[pri:9, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="OnToOneMeeting_Employee" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.OneToOneMeeting_Manager" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestSuite.OnToOneMeeting_Employee()[pri:9, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestSuite.OnToOneMeeting_Employee()[pri:9, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OnToOneMeeting_Employee -->
        <test-method signature="Finalize_Emp_Appraisal()[pri:10, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="Finalize_Emp_Appraisal" finished-at="2025-04-17T12:26:25Z" depends-on-methods="Test_Suit.SmokeTestSuite.OnToOneMeeting_Employee" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method SmokeTestSuite.Finalize_Emp_Appraisal()[pri:10, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method SmokeTestSuite.Finalize_Emp_Appraisal()[pri:10, instance:Test_Suit.SmokeTestSuite@6f15d60e] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Finalize_Emp_Appraisal -->
        <test-method signature="Remove_EmployeesFromGoalPlan()[pri:11, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:25Z" name="Remove_EmployeesFromGoalPlan" finished-at="2025-04-17T12:26:37Z" duration-ms="11848" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitUntillClickable(org.openqa.selenium.By)" because "this.explicitWait" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitUntillClickable(org.openqa.selenium.By)" because "this.explicitWait" is null
at NormalFlowForEmployee.Delete_the_PMS_Cycle.Deletion(Delete_the_PMS_Cycle.java:30)
at Test_Suit.SmokeTestSuite.Remove_EmployeesFromGoalPlan(SmokeTestSuite.java:210)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Remove_EmployeesFromGoalPlan -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:37Z" name="runOnFailure" finished-at="2025-04-17T12:26:37Z" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Remove_EmployeesFromGoalPlan status=SKIP method=SmokeTestSuite.Remove_EmployeesFromGoalPlan()[pri:11, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method signature="Remove_EmployeesFromGoalPlan()[pri:11, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:37Z" name="Remove_EmployeesFromGoalPlan" finished-at="2025-04-17T12:26:48Z" duration-ms="10717" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitUntillClickable(org.openqa.selenium.By)" because "this.explicitWait" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitUntillClickable(org.openqa.selenium.By)" because "this.explicitWait" is null
at NormalFlowForEmployee.Delete_the_PMS_Cycle.Deletion(Delete_the_PMS_Cycle.java:30)
at Test_Suit.SmokeTestSuite.Remove_EmployeesFromGoalPlan(SmokeTestSuite.java:210)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Remove_EmployeesFromGoalPlan -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:48Z" name="runOnFailure" finished-at="2025-04-17T12:26:48Z" duration-ms="262" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Remove_EmployeesFromGoalPlan status=FAILURE method=SmokeTestSuite.Remove_EmployeesFromGoalPlan()[pri:11, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method signature="DeletionOf_PMSCycleAndGoalPlan()[pri:12, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:48Z" name="DeletionOf_PMSCycleAndGoalPlan" finished-at="2025-04-17T12:26:56Z" duration-ms="7878" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitForElementVisiblity(org.openqa.selenium.By)" because "this.explicitWait" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitForElementVisiblity(org.openqa.selenium.By)" because "this.explicitWait" is null
at NormalFlowForEmployee.Delete_Goal_Plan_and_PMS_Cycle.DeletionPMSCycle(Delete_Goal_Plan_and_PMS_Cycle.java:31)
at Test_Suit.SmokeTestSuite.DeletionOf_PMSCycleAndGoalPlan(SmokeTestSuite.java:223)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeletionOf_PMSCycleAndGoalPlan -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:56Z" name="runOnFailure" finished-at="2025-04-17T12:26:56Z" duration-ms="15" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=DeletionOf_PMSCycleAndGoalPlan status=SKIP method=SmokeTestSuite.DeletionOf_PMSCycleAndGoalPlan()[pri:12, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method signature="DeletionOf_PMSCycleAndGoalPlan()[pri:12, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:26:56Z" name="DeletionOf_PMSCycleAndGoalPlan" finished-at="2025-04-17T12:27:02Z" duration-ms="6167" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitForElementVisiblity(org.openqa.selenium.By)" because "this.explicitWait" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "NormalFlowForEmployee.ExplicitWait.waitForElementVisiblity(org.openqa.selenium.By)" because "this.explicitWait" is null
at NormalFlowForEmployee.Delete_Goal_Plan_and_PMS_Cycle.DeletionPMSCycle(Delete_Goal_Plan_and_PMS_Cycle.java:31)
at Test_Suit.SmokeTestSuite.DeletionOf_PMSCycleAndGoalPlan(SmokeTestSuite.java:223)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeletionOf_PMSCycleAndGoalPlan -->
        <test-method is-config="true" signature="runOnFailure(org.testng.ITestResult)[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:27:02Z" name="runOnFailure" finished-at="2025-04-17T12:27:02Z" duration-ms="307" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=DeletionOf_PMSCycleAndGoalPlan status=FAILURE method=SmokeTestSuite.DeletionOf_PMSCycleAndGoalPlan()[pri:12, instance:Test_Suit.SmokeTestSuite@6f15d60e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runOnFailure -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Test_Suit.SmokeTestSuite@6f15d60e]" started-at="2025-04-17T12:27:02Z" name="teardown" finished-at="2025-04-17T12:27:03Z" duration-ms="534" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Test_Suit.SmokeTestSuite -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
